data/presets/building/allotment_house.json

```diff
@@ -0,0 +1,14 @@
+{
+    "icon": "maki-building",
+    "fields": [
+        "{building}"
+    ],
```

Comment:
```suggestion
```

Reply:
Do you remember which one did you use as a template? I checked a few of the existing building presets and didn't spot this issue.

Reply:
`allotment_house.json` and `outbuilding.json` - several others have this field and extra ones - it is likely necessary to repeat `{building}` when one adds extra fields, right?

(checked with `rg --context 3 "\{building\}"` using ripgrep)

Reply:
correct: when you want to add extra fields to a sub-preset the `"fields": ["{parent}", "extra"]` syntax is the way to go. But to just get the same fields in the sub-preset, it is not necessary to explicitly include the parent.